.circleci/config.yml

```diff
@@ -98,6 +98,7 @@ jobs:
       - image: trufflesuite/ganache-cli
         command: ganache-cli -i 1234 -l 9000000 -p 9545
     working_directory: ~/protocol
+    resource_class: large
```

Comment:
I wouldn't recommend making changes to the `ci` files.

Reply:
This is already outdated so resolving this issue.